DataConnectors/Exabeam/Connector_Exabeam_Syslog.json

```diff
@@ -0,0 +1,156 @@
+{
+    "id": "Exabeam",
+    "title": "Exabeam Advanced Analytics",
+    "publisher": "Exabeam",
+    "descriptionMarkdown": "The [Exabeam Advanced Analytics](https://www.exabeam.com/ueba/advanced-analytics-and-mitre-detect-and-stop-threats/) data connector provides the capability to ingest Exabeam Advanced Analytics events into Azure Sentinel. Refer to [Exabeam Advanced Analytics documentation](https://docs.exabeam.com/) for more information.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **ExabeamEvent** in queries and workbooks. [Follow steps to get this Kusto Function>](https://aka.ms/sentinel-Exabeam-parser)",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "Syslog (Exabeam)",
```

Comment:
Legend should be datatype_name which is Exabeam

Reply:
updated